runtime/queries/ada/highlights.scm

```diff
@@ -0,0 +1,178 @@
+;; highlight queries.
+;; See the syntax at https://tree-sitter.github.io/tree-sitter/using-parsers#pattern-matching-with-queries
+;; See also https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#parser-configurations
+;; for a list of recommended @ tags, though not all of them have matching
+;; highlights in neovim.
+
+[
+   "abort"
+   "abs"
+   "abstract"
+   "accept"
+   "access"
+   "all"
+   "array"
+   "at"
+   "begin"
+   "declare"
+   "delay"
+   "delta"
+   "digits"
+   "do"
+   "end"
+   "entry"
+   "exit"
+   "generic"
+   "interface"
+   "is"
+   "limited"
+   "of"
+   "others"
+   "out"
+   "pragma"
+   "private"
+   "range"
+   "synchronized"
+   "tagged"
+   "task"
+   "terminate"
+   "until"
+   "when"
+] @keyword
+[
+   "null"
+] @constant.builtin
+[
+   "aliased"
+   "constant"
+   "renames"
+] @keyword.storage
+[
+   "mod"
+   "new"
+   "protected"
+   "record"
+   "subtype"
+   "type"
+] @type.builtin
+[
+   "with"
+   "use"
+] @keyword.control.import
+[
+   "body"
+   "function"
+   "overriding"
+   "procedure"
+   "package"
+   "separate"
+] @keyword.function
+[
+   "and"
+   "in"
+   "not"
+   "or"
+   "xor"
+] @operator
+[
+   "while"
+   "loop"
+   "for"
+   "parallel"
+   "reverse"
+   "some"
+] @kewyord.control.repeat
+[
+   "return"
+] @keyword.control.return
+[
+   "case"
+   "if"
+   "else"
+   "then"
+   "elsif"
+   "select"
+] @keyword.control.conditional
+[
+   "exception"
+   "raise"
+] @keyword.control.exception
+(comment)         @comment
+(comment)         @spell       ;; spell-check comments
+(string_literal)  @string
+(string_literal)  @spell       ;; spell-check strings
```

Comment:
The `@spell` patterns can be removed. We don't support spellchecking currently but when we do we will most likely have separate queries for spell nodes (rather than being included in the highlights queries)